cmd/cli/main.go

```diff
@@ -433,17 +433,30 @@ func hostHandler(cli *vocdoniCLI) error {
 }
 
 func accountSetMetadata(cli *vocdoniCLI) error {
+	currentAccount, err := cli.api.Account("")
+	if err != nil {
+		return err
+	}
+	if currentAccount == nil {
+		return fmt.Errorf("account does not exist yet")
```

Comment:
Can we use the `ErrAccountNotExists` in order to have unified error messages ?

Reply:
Umm, this is a main.go program, its not importable by other packages. So the errors are just informative for the end user (and printed in the screen).